openpathsampling/engines/external_engine.py

```diff
@@ -257,9 +259,10 @@ def start(self, snapshot=None):
         self.start_time = time.time()
         try:
             logger.info(self.engine_command())
-            # TODO: add the ability to have handlers for stdin and stdout
             self.proc = psutil.Popen(shlex.split(self.engine_command()),
-                                     preexec_fn=os.setsid)
+                                     preexec_fn=os.setsid,
+                                     stdout=PIPE,
+                                     stderr=PIPE)
```

Comment:
Is this actually needed? I don't see these pipes used anywhere?

Reply:
This is for the `_communicate()` method. It makes it so that you can get stdout/stderr -- this was somewhat useful in debugging (told me that I wasn't generating the tpr, so I knew to look at the `grompp` command). But it seemed like `communicate` is blocking (so we just run the full length trajectory, never checking if we should kill it). So `PIPE`/`communicate` are probably *only* useful when debugging.